packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.module.ts

```diff
@@ -0,0 +1,36 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgArrayPipesModule } from 'angular-pipes';
+import { PlannerListModule, WorkItemDetailModule } from 'fabric8-planner';
+import { FeatureFlagModule } from 'ngx-feature-flag';
+import { WidgetsModule } from 'ngx-widgets';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { LoadingWidgetModule } from '../loading-widget/loading-widget.module';
+import { WorkItemBarchartModule } from '../work-item-widget/work-item-barchart/work-item-barchart.module';
+import { SpaceSettingsWidgetComponent } from './space-settings-widget.component';
+import { EditSpaceDescriptionWidgetModule } from '../edit-space-description-widget/edit-space-description-widget.module';
+import { AddCollaboratorsDialogModule } from '../../space/settings/collaborators/add-collaborators-dialog/add-collaborators-dialog.module';
+
+@NgModule({
+  imports: [
+    RouterModule,
+    ModalModule,
+    CommonModule,
+    FeatureFlagModule,
+    FormsModule,
+    LoadingWidgetModule,
+    WidgetsModule,
+    PlannerListModule,
+    NgArrayPipesModule,
+    WorkItemBarchartModule,
+    WorkItemDetailModule,
+    AddCollaboratorsDialogModule,
+    EditSpaceDescriptionWidgetModule,
```

Comment:
Why are you importing ```FormsModule```, ```WidgetsModule```, ```PlannerListModule```, ```WorkItemBarchartModule```,
and ```WorkItemDetailModule```? These modules have been not used in the component.

Reply:
removed unused modules from imports